src/panels/config/cloud/alexa/cloud-alexa.ts

```diff
@@ -346,20 +368,32 @@ class CloudAlexa extends LitElement {
   private async _openAliasesSettings(ev) {
     ev.stopPropagation();
     const entityId = ev.target.entityId;
-    const entry = await getExtendedEntityRegistryEntry(this.hass, entityId);
+    const entry = this._entries![entityId];
     if (!entry) {
       return;
     }
     showEntityAliasesDialog(this, {
       entity: entry,
       updateEntry: async (updates) => {
-        await updateEntityRegistryEntry(this.hass, entry.entity_id, updates);
+        const { entity_entry } = await updateEntityRegistryEntry(
+          this.hass,
+          entry.entity_id,
+          updates
+        );
+        this._entries![entity_entry.entity_id] = entity_entry;
       },
     });
   }
 
   private async _fetchData() {
     const entities = await fetchCloudAlexaEntities(this.hass);
+    this._entries = await getExtendedEntityRegistryEntries(
+      this.hass,
+      entities
+        .filter((ent) => this.hass.entities[ent.entity_id])
```

Comment:
Do we need to filter this? Will core not handle this?

Reply:
It will return null for entities without unique id. It's to reduce the request and response payload.